File: tests/golden/fixtures/generator/hbs/lua-helper/addons/lua/generator/common/helpers/echo.lua
-- Echo helper used in golden tests; keeps output stable across engines.
-- Uses normalize_args from _utils.lua (loaded before helper files).

return function(...)
    local list = normalize_args(...)
    local value
    if list.n > 0 and list[1] ~= nil then
        value = list[1]
    else
        value = ""
    end
    return "lua:" .. tostring(value)
end

File: tests/golden/fixtures/generator/hbs/js-helper/addons/js/generator/common/helpers/echo.js
// Echo helper used in golden tests; keeps output stable across engines.
// Uses normalize_args from _utils.js (loaded before helper files).

function echo()
{
    var args = normalize_args(arguments);
    var value = args.length > 0 ? args[0] : '';
    return 'js:' + value;
}
